### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by DubbleClick**

### CODE REVIEW

File: GWToolboxdll/Windows/Hotkeys.cpp [hunk 1]
```diff
@@ -1511,12 +1511,10 @@ void HotkeyTarget::Execute()
         break;
     default:
         Log::ErrorW(L"Unknown target type %d", type);
-        delete[] message;
         return;
     }
     GW::GameThread::Enqueue([message]() {
         GW::Chat::SendChat('/', message);
-        delete[] message;
```
Comment (3vcloud) [inline GWToolboxdll/Windows/Hotkeys.cpp:1519]: This is why its necessary

Comment (DubbleClick) [reply GWToolboxdll/Windows/Hotkeys.cpp:1519]: copies message by value with default capture

Comment (3vcloud) [reply GWToolboxdll/Windows/Hotkeys.cpp:1519]: Right you are, but why copy?

Comment (DubbleClick) [reply GWToolboxdll/Windows/Hotkeys.cpp:1519]: compiler optimises it away into a move anyway, pretty sure

memory safety > minimal performance gain (though with optimisations most likely a performance penalty for the heal allocation anyway)

**Pull request merged**